oceanmesh/mesh_generator.py

```diff
@@ -696,17 +696,41 @@ def _stereo_distortion_dist(lat):
     return res
 
 
+def _parametric(lat):
+    ones = np.ones(lat.shape)
```

Comment:
Thanks for this! Could you explain a bit more how you arrived at these equations?

Reply:
Yes, I have tried to match the edge length distribution I observed close to the poles, I detail it more in the issue [here](https://github.com/CHLNDDEV/oceanmesh/issues/75#issuecomment-2040207824)
Although I have been taking the problem on the wrong end.
I'm gonna explain why in the same issue.